nengo_gui/examples/tutorial/02-two-neurons.py

```diff
@@ -19,7 +19,7 @@
     stim = nengo.Node(0)
     ens = nengo.Ensemble(n_neurons=2, dimensions=1, seed=5)
     nengo.Connection(stim, ens)
-    
-    
+
+
     import time
-    slowdown = nengo.Node(lambda t: time.sleep(0.001)) 
\ No newline at end of file
+    slowdown = nengo.Node(lambda t: time.sleep(0.001))
```

Comment:
Fair enough.  Let's add in the comment that you've got here, and include the comment and the code in examples 00, 01, and 02.